examples/worker/worker_util.py

```diff
@@ -1,44 +1,13 @@
 from conductor.client.automator.task_handler import TaskHandler
-from conductor.client.worker.worker import Worker
-from conductor.client.worker.worker_interface import WorkerInterface
-
 from examples.api import api_util
-from examples.worker import workers
 
 
 def start_workers() -> TaskHandler:
     task_handler = TaskHandler(
-        workers=[
-            create_worker_get_user_info(),
-            create_worker_send_email(),
-            create_worker_send_sms()
-        ],
-        configuration=api_util.get_configuration()
+        workers=[],
+        configuration=api_util.get_configuration(),
```

Comment:
an we move this into a simple function instead of `api_util`?

Reply:
Got it, sending a PR with an attempt to allow `None` for `configuration`. It should research for ENVs and fall back to a fixed value otherwise. In practice, the process will be the same but hidden within the SDK.